vision/util/capture_airsim_image.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tool to capture scene (analogous to color images from realsense)
+and depth images from airsim
+
+A commented snippet of code is also included to demonstrate how to
+translate the images into a numpy array for further processing
+
+To capture/save images, start and run the airsim. Then, navigate to
+the location of this program in terminal, and run using
+"python capture_airsim_image.py". Enter q to capture an image, or e to exit.
+"""
+import airsim
```

Comment:
This file looks good but lets add a file that templates Camera that does the same thing and we are good to go.